models/experimental/functional_yolox_m/demo/demo_ttnn.py

```diff
@@ -0,0 +1,677 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+from plumbum import ProcessLineTimedOut
+from models.experimental.functional_yolox_m.reference.csp_darknet import CSPDarknet
+import torch
+import torchvision
+import torch.nn as nn
+import cv2
+import numpy as np
+from ttnn.model_preprocessing import preprocess_model
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from models.utility_functions import skip_for_wormhole_b0
+
+
+from models.experimental.functional_yolox_m.reference.yolox_m import YOLOX
+from models.experimental.functional_yolox_m.tt.ttnn_yolox import TtYOLOX
+from models.experimental.functional_yolox_m.reference.dark2 import Focus, Dark2
+from models.experimental.functional_yolox_m.tt.ttnn_dark2 import TtFocus, TtDark2
+from models.experimental.functional_yolox_m.tt.ttnn_yolopafpn import TtYOLOPAFPN
+from models.experimental.functional_yolox_m.tt.ttnn_cspdarknet import TtCSPDarknet
+
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_d2 as D2
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_d3 as D3
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_d4 as D4
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_d5 as D5
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_cspdarknet as cspdarknet
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_yolopafpn as fpn
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_yolohead as head
+import ttnn
+
+cls_names = (
+    "person",
+    "bicycle",
+    "car",
+    "motorcycle",
+    "airplane",
+    "bus",
+    "train",
+    "truck",
+    "boat",
+    "traffic light",
+    "fire hydrant",
+    "stop sign",
+    "parking meter",
+    "bench",
+    "bird",
+    "cat",
+    "dog",
+    "horse",
+    "sheep",
+    "cow",
+    "elephant",
+    "bear",
+    "zebra",
+    "giraffe",
+    "backpack",
+    "umbrella",
+    "handbag",
+    "tie",
+    "suitcase",
+    "frisbee",
+    "skis",
+    "snowboard",
+    "sports ball",
+    "kite",
+    "baseball bat",
+    "baseball glove",
+    "skateboard",
+    "surfboard",
+    "tennis racket",
+    "bottle",
+    "wine glass",
+    "cup",
+    "fork",
+    "knife",
+    "spoon",
+    "bowl",
+    "banana",
+    "apple",
+    "sandwich",
+    "orange",
+    "broccoli",
+    "carrot",
+    "hot dog",
+    "pizza",
+    "donut",
+    "cake",
+    "chair",
+    "couch",
+    "potted plant",
+    "bed",
+    "dining table",
+    "toilet",
+    "tv",
+    "laptop",
+    "mouse",
+    "remote",
+    "keyboard",
+    "cell phone",
+    "microwave",
+    "oven",
+    "toaster",
+    "sink",
+    "refrigerator",
+    "book",
+    "clock",
+    "vase",
+    "scissors",
+    "teddy bear",
+    "hair drier",
+    "toothbrush",
+)
+
+
+def vis(img, boxes, scores, cls_ids, conf=0.5, class_names=None):
```

Comment:
Can we have a file `demo_utils.py` and keep all the utility methods in that file?